tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+collect_ignore = []
+if sys.version_info[0] > 2:
+    collect_ignore.append("test_mod_apache.py")
+    collect_ignore.append("test_lib_smsprovider.py")
```

Comment:
The sms test is excluded?
Is there a problem, why this test would not run?

Reply:
We currently have no working `smpp` library for Python 3 thus the test won't even load.
This was a mere example of how to exclude tests for specific Python versions. Currently there are several tests failing with Python 3.5/3.6, i have just not added all of them here.

Reply:
OK, I see. These tests are only excluded for python ver > 2.